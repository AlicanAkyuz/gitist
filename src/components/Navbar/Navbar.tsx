import React, { FunctionComponent } from 'react';

const Navbar: FunctionComponent = () => <header className="navbar">Gitist</header>;

export default Navbar;
