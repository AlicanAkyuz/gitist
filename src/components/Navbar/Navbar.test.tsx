import React from 'react';
import { shallow } from 'enzyme';

import Navbar from './Navbar';

it('Navbar corrently renders a header with title', () => {
  const wrapper = shallow(<Navbar />);
  expect(wrapper).toMatchSnapshot();
});
