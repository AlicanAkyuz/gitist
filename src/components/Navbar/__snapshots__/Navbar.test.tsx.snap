// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar corrently renders a header with title 1`] = `
<header
  className="navbar"
>
  Gitist
</header>
`;
